README.md

```diff
@@ -68,7 +68,19 @@ engines:
 Note that you have the update the YAML structure under the `languages` key to
 the Hash type to support extra configuration.
 
+### Excluding files and directories
+
+As with any other Codeclimate engine, you can exclude certain files or directories from being analyzed. For more information, [see *Exclude paths for specific engines* on our documentation][exclude-files-engine].
```

Comment:
That makes sense. I tried to get the link to be as descriptive as possible for a17n, while emphasizing the subtitle of the target document.